ragna/assistants/_anthropic.py

```diff
@@ -18,26 +18,27 @@ def _extra_requirements(cls) -> list[Requirement]:
     def display_name(cls) -> str:
         return f"Anthropic/{cls._MODEL}"
 
-    def _instructize_prompt(self, prompt: str, sources: list[Source]) -> str:
-        # See https://docs.anthropic.com/claude/docs/introduction-to-prompt-design#human--assistant-formatting
+    def _instructize_system_prompt(self, sources: list[Source]) -> str:
+        # See https://docs.anthropic.com/claude/docs/system-prompts
         instruction = (
-            "\n\nHuman: "
-            "Use the following pieces of context to answer the question at the end. "
-            "If you don't know the answer, just say so. Don't try to make up an answer.\n"
+            "Use the following documents to answer the prompt. "
+            "If you don't know the answer, just say so. Don't try to make up an answer."
+            "Only use the included documents below to generate the answer.\n"
         )
         instruction += "\n\n".join(source.content for source in sources)
```

Comment:
Instead of just putting the sources in plain text, we should follow their guidelines and use HTML tags: https://docs.anthropic.com/claude/docs/long-context-window-tips#structuring-long-documents